tests/src/main/kotlin/se/ansman/kotshi/ContainsComplexlyQualifiedString.kt

```diff
@@ -0,0 +1,44 @@
+package se.ansman.kotshi
+
+import com.squareup.moshi.JsonQualifier
+import kotlin.reflect.KClass
+
+// TODO: Elements.getElementValuesWithDefaults(AnnotationMirror) does not find elements with Java keyword names
+// TODO: like `class`, boolean, switch, enum!
```

Comment:
Why is this?
When I had `@JsonQualifier annotation class WithBooleanElement(val boolean: Boolean)`, my call to `Elements.getElementValuesWithDefaults(AnnotationMirror)` here https://github.com/ansman/kotshi/pull/62/files#diff-ef2fb2b136b87eda31462cf64df5c75cR217 returned an empty map!